### PR TITLE
[Reviewer: Ellie] Create default files, plugins side

### PR DESCRIPTION
**Pull request by johadalin**
This is the plugins side of creating default files and inserting them into etcd if no key is present already.

Tested live, and correct files are created. Also used `curl http://127.0.0.1:4000/v2/keys/clearwater/single_site/configuration/<file key>` to check that the files had been added to etcd correctly.

### CODE REVIEW

File: clearwater_config_manager/dns_config_plugin.py [hunk 1]
```diff
@@ -36,6 +36,11 @@
 
 _log = logging.getLogger("dns_config_plugin")
 _file = "/etc/clearwater/dns_config"
+_default_value = """\
+{
```
Comment (eleanor-merry) [inline clearwater_config_manager/dns_config_plugin.py:40]: I take it that this indentation makes it to the file on disk?

File: clearwater_config_manager/shared_config_plugin.py [hunk 2]
```diff
@@ -68,5 +72,10 @@ def on_config_changed(self, value, alarm):
             safely_write(_file, value)
             run_command("/usr/share/clearwater/clearwater-queue-manager/scripts/modify_nodes_in_queue add apply_config")
 
+    def on_creating_etcd_key(self, value):
+        if value != _default_value:
+            _log.debug("Created non-default config key in etcd. Adding node to restart queue")
```
Comment (eleanor-merry) [inline clearwater_config_manager/shared_config_plugin.py]: Can you either remove this log, or add equivalent logs to the other plugins as well. I wonder if the log should also include the value (and parse down the log you've got).

File: clearwater_config_manager/shared_config_plugin.py [hunk 3]
```diff
@@ -39,6 +39,7 @@
 
 _log = logging.getLogger("shared_config_plugin")
 _file = "/etc/clearwater/shared_config"
+_default_value = "### Create Shared Config Here ###"
```
Comment (eleanor-merry) [inline clearwater_config_manager/shared_config_plugin.py]: Can this include a warning that the shared config is currently missing?

**Review by eleanor-merry — APPROVED** [review]
Sure (although I'm not sure about the on_creating_etcd_key method)

File: clearwater_config_manager/dns_config_plugin.py (hunk 1 above)
Comment (johadalin) [reply clearwater_config_manager/dns_config_plugin.py:40]: Yes

**Pull request merged**